org.eclipse.lsp4e/src/org/eclipse/lsp4e/FileBufferListenerAdapter.java

```diff
@@ -15,7 +15,7 @@
 import org.eclipse.core.filebuffers.IFileBufferListener;
 import org.eclipse.core.runtime.IPath;
 
-class FileBufferListenerAdapter implements IFileBufferListener {
+public class FileBufferListenerAdapter implements IFileBufferListener {
```

Comment:
Maybe instead of now exposing FileBufferListenerAdapter as part of the plugin's API we should move it to the internal package.